components/https_airbyte_com/https_airbyte_com.app.mjs

```diff
@@ -1,11 +1,86 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "https_airbyte_com",
-  propDefinitions: {},
+  propDefinitions: {
+    name: {
+      type: "string",
+      label: "Workspace name",
+      description: "Name of the workspace",
+    },
+    workspaceId: {
+      type: "string",
+      label: "Workspace ID",
+      description: "ID of the workspace",
+      async options() {
+        const { data } = await this.listWorkspaces();
+
+        return data.map(({
+          workspaceId, name,
+        }) => ({
+          label: name,
+          value: workspaceId,
+        }));
+      },
+    },
+    includeDeleted: {
+      type: "boolean",
+      label: "Include deleted",
+      description: "Include deleted workspaces in the list",
+      optional: true,
+    },
```

Comment:
Was the prop `includeDeleted` meant to be included in `list-workspaces`?

Reply:
Yes, you can look here https://reference.airbyte.com/reference/listworkspaces